Dockerfile

```diff
@@ -29,4 +29,5 @@ RUN go generate && go build -o /usr/bin/spark-operator
 
 FROM gcr.io/ynli-k8s/spark:v2.3.0
```

Comment:
This will also require an image built using the Dockerfile in the latest spark/master, right?